packages/react/src/MediaFile.tsx

```diff
@@ -83,11 +83,11 @@ export function MediaFile({
       );
     }
     default: {
-      const typenameMissingMessage = `<MediaFile /> requires the '__typename' property to exist on the 'data' prop in order to correctly render the correct component for this media. Rendering 'null' by default`;
+      const typenameMissingMessage = `<MediaFile /> requires the '__typename' property to exist on the 'data' prop in order to correctly render the correct component for this media.`;
```

Comment:
The first 'correctly' isn't needed. 

```suggestion
      const typenameMissingMessage = `<MediaFile /> requires the '__typename' property to exist on the 'data' prop in order to render the matching sub-component for this type of media.`;
```

The other suggestions below I could go either way on.